PhpAmqpLib/Wire/IO/SocketIO.php

```diff
@@ -41,9 +41,9 @@ class SocketIO extends AbstractIO
      * @param float $read_timeout
      * @param bool $keepalive
      * @param float|null $write_timeout if null defaults to read timeout
-     * @param int $heartbeat
+     * @param int $heartbeat how often to send heartbeat. 0 means off
      */
-    public function __construct($host, $port, $read_timeout, $keepalive = false, $write_timeout = null, $heartbeat)
+    public function __construct($host, $port, $read_timeout, $keepalive = false, $write_timeout = null, $heartbeat = 0)
```

Comment:
@michaelklishin 
I believe this is a BC break as the functionality will change when upgrading.
Unfortunately, even if the recommended default should be to have heartbeat enabled and set to 60, for this PHP extension the default was to have it disabled.
I believe the original authors did this because it is not very easy to handle heartbeat logic in PHP, or at least it's not easy when not using coroutines and async behaviours.

I believe it is not safe to turn on this in a minor version as it can break library usage.